### DISCUSSION TITLE
The Event Game End event does not work.

### DISCUSSION THREAD

**Discussion by rimuruchaos** [roflmuffin/CounterStrikeSharp#692] [Q&A]
The Event Game End event does not work. when the last round ends, I want to do some action, but the event does not work what to do or which event to use

mycode
![image](https://github.com/user-attachments/assets/74b3f902-d33d-4e6e-8e9e-3bc35b9b2356)

![image](https://github.com/user-attachments/assets/533a229a-833e-451f-887f-0747a13b75f0)

**Comment by KillStr3aK** [comment]
Hey!

if an event is not called then even the game does not call it, you can use the win panel one for game end
maybe give a try to `cs_win_panel_match`
